### PR TITLE
pipeline release functionality

### PR DESCRIPTION
**Pull request by wesleymccollam**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Visit https://goreleaser.com for documentation on how to customize this
+# behavior.
+before:
```
Comment (henryrecker-pingidentity) [inline .goreleaser.yml:3]: Can we remove this?

File: .goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      - "${signature}"
+      - "--detach-sign"
+      - "${artifact}"
+release:
+  extra_files:
+    - glob: 'terraform-registry-manifest.json'
+      name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'
+  # If you want to manually examine the release before its live, uncomment this line:
+  # draft: true
+changelog:
```
Comment (henryrecker-pingidentity) [inline .goreleaser.yml:59]: How do we generate the changelog locally? Run goreleaser by hand?

File: terraform-registry-manifest.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (henryrecker-pingidentity) [inline terraform-registry-manifest.json:1]: What is this file for? Should we have this file yet if we aren't publishing yet?

Comment (wesleymccollam) [reply terraform-registry-manifest.json:1]: used in the checksum of the .goreleaser file

`checksum:
  extra_files:
    - glob: 'terraform-registry-manifest.json'
      name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'`

Comment (wesleymccollam) [reply terraform-registry-manifest.json:1]: removed for now, will add back when on registry

**Review by henryrecker-pingidentity — APPROVED** [review]

**Pull request merged**